src/routes/RoutesConstants.jsx

```diff
@@ -0,0 +1,18 @@
+import { Route, Routes } from 'react-router-dom';
+import PrivateRoutes from 'routes/PrivateRoutes';
+import { Leaderboard } from 'container/leaderboard/leaderboard';
+import Login from 'container/login/login';
+import MainMenu from 'container/main-menu/mainMenu';
+export const RoutesConstants = () => {
+  return (
+    <div>
+      <Routes>
+        <Route element={<PrivateRoutes />}>
```

Comment:
Create a contants file in routes folder and declare these paths as a const

Reply:
main menu path should b '/' not 'mainMenu'.
For paths we follow kebab case like main-menu